azure-pipelines.yml

```diff
@@ -0,0 +1,235 @@
+# Azure DevOps pipeline to build, check source codes, run tests, and deploy.
+#
+# To make Danger JS run on a pull request you need to add the following pipeline 
+# variable and set it with a GitHub access token (scope public_repo); otherwise 
+# set its value to 'skip' without marking it secret: 
+# - DANGER_GITHUB_API_TOKEN
+#
+# To enable the deployment in any environment you need to configure the following 
+# global variable otherwise all the deployment jobs will be always skipped:
+# - DO_DEPLOY = true
+# you also need to configure the following additional parameters based on the 
+# environment(s) to update when running the pipeline:
+# - PRODUCTION_ENABLE_DEPLOY = true
+# - TEST_ENABLE_DEPLOY = true
+#
+# The following parameter and variables must also be set to run the deployment:
+# - PRODUCTION_DEPLOY_TYPE:
+#    -- 'deployToStagingSlot': deploy to 'staging' slot 
+#    -- 'deployToProductionSlot' (default): deploy to 'production' slot
+#    -- 'deployToStagingSlotAndSwap': deploy to 'staging' slot and then swap 
+# - PRODUCTION_AZURE_SUBSCRIPTION
+# - PRODUCTION_RESOURCE_GROUP_NAME
+# - PRODUCTION_APP_NAME
+#
+# - TEST_DEPLOY_TYPE:
+#      -- 'deployToTestSlot' (default): deploy to 'test' slot 
+#      -- 'deployToProductionSlot': deploy to 'production' slot (TEST env)
+# - TEST_AZURE_SUBSCRIPTION
+# - TEST_RESOURCE_GROUP_NAME
+# - TEST_APP_NAME
+# 
+# Please note that the deployment to a slot (e.g. 'staging') could automatically
+# trigger the swapping with 'production' slot if "auto swap" has been enabled in
+# the App service configuration. This also means that if the "auto swap" is on and 
+# you select 'deployToStagingSlotAndSwap' to swap 'staging' and 'production' slots
+# after deploying to 'staging', the swap task in this pipeline will probably fail
+# because conflicting with another (automatic) swap operation in progress.
+# 
+
+variables:
+  NODE_VERSION: '12.14.1'
```

Comment:
@BurnedMarshal can you check NODE_VERSION?

Reply:
I cannot see on [Azure documentation](https://docs.microsoft.com/en-us/azure/azure-functions/functions-reference-node?tabs=v2#node-version) any limitation on Node 12 version. Maybe we can use the latest release `12.19.0`.
cc @gunzip @uolter

Reply:
We need to set the same version also here

https://github.com/pagopa/io-infrastructure-live-new/blob/175476380-web-app-dashboard-support/prod/westeurope/support/functions_backoffice/function_app/terragrunt.hcl#L59

Reply:
@pasqualedevita .... updated to 12.19.0 in terragrunt

Reply:
ok setted to `12.19.0`